js/src/ui/Form/InputChip/inputChip.css

```diff
@@ -17,10 +17,10 @@
 
 .chip {
   & > svg {
-    width: 1.2rem !important;
-    height: 1.2rem !important;
+    width: 1rem !important;
+    height: 1rem !important;
     margin: initial !important;
     margin-right: 4px !important;
-    padding: 4px 0 !important;
+    padding: 2px 0 !important;
```

Comment:
Using absolute/root-relative values will make the layout break if we change the font size. If possible, use `em`.